docs/monitoring.md

```diff
@@ -95,6 +95,44 @@ The history server can be configured as follows:
   </tr>
 </table>
 
+### Applying compaction of old event log files
+
+A long-running streaming application can bring a huge single event log file which may cost a lot to maintain and
```

Comment:
I assume this works for any application, not just streaming?  If so perhaps we change the wording a little to say for large application, and give streaming as example. I used to see long running graph processing applications create huge log files as well. 
 I also think we should describe what compaction is here up front because if I just read this, I would assume its some sort of compression, but really its throwing away parts of the files.

Reply:
> I assume this works for any application, not just streaming?

You're right. I focused too much on the target of compaction which is most likely streaming application, but in this sentence it's not only for `streaming`.

> I also think we should describe what compaction is here up front

Uh, actually we don't have explicit section for rolling event log, hence I feel it's good to explain what's rolling event log first, and what is "compaction". Otherwise maybe good to have individual section for rolling event log?